frontend/src/components/maps/leaflet/InfoSlideOut/BuildingAttributes.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```

Comment:
My ticket just says for parcels/land, but I added this as a sort of placeholder so at least there's something